FOC Devnet scenarios run report (frontier)

The **frontier** scenarios run **failed ❌**.
See [the workflow run](https://github.com/FilOzone/foc-devnet/actions/runs/26705747231) for details.


# Scenarios Tests 

| Description | Data                                                                |
|-------------| ------------------------------------------------------------------- |
| Type        | **frontier**                                                       |
| Date        | 31-May-2026 07:23:51 GMT +0                                                               |
| Status      | PASS ✅:**3**, FAIL 🟥:**2**, Total:5 |
| CI run      | [https://github.com/FilOzone/foc-devnet/actions/runs/26705747231](https://github.com/FilOzone/foc-devnet/actions/runs/26705747231)                                                        |

## Versions info
```
foc-devnet version: not available
```

## Tests summary
<details>
<summary>✅ <b>test_containers</b> - PASS (0s)</summary>

```
[ OK ] container foc-20260531T0711_TizzyBear-lotus is running
[ OK ] container foc-20260531T0711_TizzyBear-lotus-miner is running
[ OK ] container foc-20260531T0711_TizzyBear-curio-1 is running
[ OK ] container foc-20260531T0711_TizzyBear-curio-2 is running
```
</details>

<details>
<summary>✅ <b>test_basic_balances</b> - PASS (0s)</summary>

```
[ OK ] cast is installed
[ OK ] at least one user exists
[ OK ] USER_1 FIL balance > 0
[ OK ] USER_1 USDFC balance > 0
[ OK ] USER_2 FIL balance > 0
[ OK ] USER_2 USDFC balance > 0
[ OK ] USER_3 FIL balance > 0
[ OK ] USER_3 USDFC balance > 0
```
</details>

<details>
<summary>❌ <b>test_storage_e2e</b> - FAIL (22s)</summary>

```
[FAIL] pnpm install (exit=1) Scope: all 9 workspace projects
Progress: resolved 1, reused 0, downloaded 0, added 0
Progress: resolved 67, reused 0, downloaded 61, added 0
Progress: resolved 82, reused 0, downloaded 78, added 0
Progress: resolved 85, reused 0, downloaded 83, added 0
Progress: resolved 86, reused 0, downloaded 85, added 0
Progress: resolved 172, reused 0, downloaded 155, added 0
Progress: resolved 270, reused 0, downloaded 232, added 0
Progress: resolved 337, reused 0, downloaded 271, added 0
Progress: resolved 401, reused 0, downloaded 338, added 0
Progress: resolved 519, reused 0, downloaded 425, added 0
Progress: resolved 629, reused 0, downloaded 510, added 0
Progress: resolved 720, reused 0, downloaded 582, added 0
Progress: resolved 807, reused 0, downloaded 624, added 0
Progress: resolved 913, reused 0, downloaded 717, added 0
Progress: resolved 971, reused 0, downloaded 777, added 0
Progress: resolved 1035, reused 0, downloaded 818, added 0
 WARN  Request took 10704ms: https://registry.npmjs.org/es-toolkit
Progress: resolved 1091, reused 0, downloaded 919, added 0
 WARN  Request took 12060ms: https://registry.npmjs.org/@typescript-eslint%2Fparser
Progress: resolved 1185, reused 0, downloaded 1014, added 0
 WARN  Request took 10211ms: https://registry.npmjs.org/playwright-core
Progress: resolved 1278, reused 0, downloaded 1083, added 0
/tmp/synapse-sdk-zb0_k3nd/synapse-sdk/docs:
 ERR_PNPM_TRUST_DOWNGRADE  High-risk trust downgrade for "tinyexec@1.2.2" (possible package takeover)

This error happened while installing the dependencies of astro@6.3.7

Trust checks are based solely on publish date, not semver. A package cannot be installed if any earlier-published version had stronger trust evidence. Earlier versions had trusted publisher, but this version has provenance attestation. A trust downgrade may indicate a supply chain incident.
[ OK ] git is installed
[ OK ] node is installed
[ OK ] pnpm is installed
[ OK ] clone synapse-sdk
[ OK ] checkout master HEAD
[INFO] synapse-sdk commit: 000d27622c36ab3989f4a6cf9bff98967ac522e7
```
</details>

<details>
<summary>✅ <b>test_multi_copy_upload</b> - PASS (69s)</summary>

```
[ OK ] node is installed
[ OK ] npm is installed
[ OK ] npm init
[ OK ] pin filecoin-pin dependencies
[ OK ] npm install
[INFO] Patched @filoz/synapse-core streaming upload (stripped 1 Content-Length line(s))
[INFO] Creating random file (20971520 bytes)
[ OK ] random_file created with exact size 20971520 bytes
[INFO] Running filecoin-pin multi-copy upload script against devnet
[INFO] filecoin-pin add attempt 1 (default multi-copy)
[INFO] Verified retrieval URL 1: http://host.docker.internal:5721/ipfs/bafybeicscvnp3dhanfl3yrgdavs35em6i4ycrkon5gvoggfnn6gfqdfj4q
[INFO] Verified retrieval URL 2: http://host.docker.internal:5725/ipfs/bafybeicscvnp3dhanfl3yrgdavs35em6i4ycrkon5gvoggfnn6gfqdfj4q
```
</details>

<details>
<summary>❌ <b>test_caching_subsystem</b> - FAIL (10s)</summary>

```
[FAIL] pnpm install (exit=1) Scope: all 9 workspace projects
Progress: resolved 1, reused 0, downloaded 0, added 0
Progress: resolved 86, reused 86, downloaded 0, added 0
Progress: resolved 187, reused 178, downloaded 0, added 0
Progress: resolved 384, reused 348, downloaded 0, added 0
Progress: resolved 675, reused 525, downloaded 0, added 0
Progress: resolved 994, reused 793, downloaded 0, added 0
Progress: resolved 1185, reused 988, downloaded 0, added 0
/tmp/synapse-sdk-cache-l1saae5r/synapse-sdk/docs:
 ERR_PNPM_TRUST_DOWNGRADE  High-risk trust downgrade for "tinyexec@1.2.2" (possible package takeover)

This error happened while installing the dependencies of astro@6.3.7

Trust checks are based solely on publish date, not semver. A package cannot be installed if any earlier-published version had stronger trust evidence. Earlier versions had trusted publisher, but this version has provenance attestation. A trust downgrade may indicate a supply chain incident.
[ OK ] git is installed
[ OK ] node is installed
[ OK ] pnpm is installed
[INFO] Run index: 1, seeds: small=43, large=86
[INFO] cqlsh version: cqlsh 6.2.0
[INFO] Yugabyte cassandra port: localhost:5705
[INFO] Initial row count = 0
[ OK ] clone synapse-sdk
[ OK ] checkout master HEAD
[INFO] synapse-sdk commit: 000d27622c36ab3989f4a6cf9bff98967ac522e7
```
</details> [Action Run link](https://github.com/FilOzone/foc-devnet/actions/runs/26705747231)

--- a/.github/workflows/ci_nightly.yml
+++ b/.github/workflows/ci_nightly.yml
@@ -30,7 +30,7 @@ jobs:
       matrix:
         include:
           - name: stability
-            init_flags: "--curio latesttag:pdp/v* --filecoin-services latesttag:v*"
+            init_flags: "--curio gittag:v1.28.2 --filecoin-services latesttag:v*"
             issue_label: scenarios-run-stability
             issue_title: "FOC Devnet scenarios run report (stability)"
           - name: frontier

--- a/scenarios/test_multi_copy_upload.py
+++ b/scenarios/test_multi_copy_upload.py
@@ -246,7 +246,7 @@ def run():
                 "pkg",
                 "set",
                 "type=module",
-                "dependencies.filecoin-pin=0.22.3",
+                "dependencies.filecoin-pin=1.0.1",
                 "dependencies.multiformats=13.4.2",
             ],
             label="pin filecoin-pin dependencies",

--- a/src/config.rs
+++ b/src/config.rs
@@ -285,15 +285,15 @@ impl Default for Config {
             port_range_count: 100,
             lotus: Location::GitTag {
                 url: "https://github.com/filecoin-project/lotus.git".to_string(),
-                tag: "v1.35.0".to_string(),
+                tag: "v1.36.0".to_string(),
             },
-            curio: Location::GitCommit {
+            curio: Location::GitTag {
                 url: "https://github.com/filecoin-project/curio.git".to_string(),
-                commit: "60f77a618eee24cd3482be5fea545e01f26052a4".to_string(),
+                tag: "v1.28.2".to_string(),
             },
             filecoin_services: Location::GitTag {
                 url: "https://github.com/FilOzone/filecoin-services.git".to_string(),
-                tag: "v1.2.0".to_string(),
+                tag: "v1.3.0".to_string(),
             },
             multicall3: Location::GitTag {
                 url: "https://github.com/mds1/multicall3.git".to_string(),

